modules/marketsPage/components/BackgroundGradient.tsx

```diff
@@ -27,13 +29,13 @@ const BackgroundGradient: FunctionComponent<BackgroundGradientProps> = ({}) => {
                     </g>
                 </svg>
             </div>
-            {chain.chain.id === customChains.bsc.id && (
+            {chainID === customChains.bsc.id && (
```

Comment:
better to rewrite to new component

and using if else or switch case. 

it's more readable

Reply:
fixed sir, please kindly check again